### PR TITLE
add needs-trage activity

### PR DESCRIPTION
**Pull request by xiangyan99**
# Description

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.

# All SDK Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes]**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md).**

## General Guidelines and Best Practices
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md##building-and-testing)
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

**Review by jsquire — APPROVED** [review]
I'm guessing no response from our support request?  That makes me sad.  

If we're going to start the slow port of bot rules over to actions, we should probably huddle up at some point and talk over a common approach for organizing/naming so that we can build on this work as we centralize.

File: .github/workflows/label-needs-triage.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+name: Add needs-triage label to new issue
```
Comment (jsquire) [inline .github/workflows/label-needs-triage.yml:1]: Should we remove the Fabric Bot rule?   Probably no harm in having two things apply the label... I hope?

Comment (xiangyan99) [reply .github/workflows/label-needs-triage.yml:1]: I think this rule is straight forward and I wanted to use this one as an experiment.

If you don't have objections, I will keep both of them so we are consistent with other repos.

Comment (jsquire) [reply .github/workflows/label-needs-triage.yml:1]: No objections from me.

**Review by scbedd — APPROVED** [review]

**Pull request merged**